File: TerriaMVC/wwwroot/js/DTO/GenerateAOIDTO.js
class GenerateAOIDTO {
    constructor(idInstrumentos, defaultContent, isForTop, isForTopConsolidado, isForBotConsolidado) {
        this.IdInstrumentos = idInstrumentos;
        this.DefaultContent = defaultContent;
        this.IsForTop = isForTop;
        this.IsForTopConsolidado = isForTopConsolidado;
        this.IsForBotConsolidado = isForBotConsolidado;
    }
    async readValues() {
        console.log("this.IdInstrumentos: " + this.IdInstrumentos);
        console.log("this.DefaultContent: " + this.DefaultContent)
        console.log("this.IsForTop: " + this.IsForTop)
        console.log("this.IsForTopConsolidado: " + this.IsForTopConsolidado)
        console.log("this.IsForBotConsolidado: " + this.IsForBotConsolidado)
    }
}

File: TerriaMVC/DTO/GenerateAOIDTO.cs
namespace TerriaMVC.DTO
{
    public class GenerateAOIDTO
    {
        public int[] idInstrumentos { get; set; }
        public bool defaultContent { get; set; } //Mostrar la primera pestaña con el gráfico y los botónes de los instrumentos del tipo de instrumento.
        public bool? isForTop { get; set; }
        public bool? isForTopConsolidado { get; set; }
        public bool? isForBotConsolidado { get; set; }
    }
}
